lib/svelte/rest_client.rb

```diff
@@ -18,9 +18,10 @@ class << self
       #   an exception will be raised
       #
       # @return [Faraday::Response] http response from the service
-      def call(verb:, url:, params: {}, options: {})
+      def call(verb:, url:, params: {}, options: {}, headers: nil)
         connection.send verb, url, params do |request|
           request.options.timeout = options[:timeout] if options[:timeout]
+          request.headers = headers unless headers.nil?
```

Comment:
Personally I'd set headers to an empty hash so this isn't needed.

( [this](https://github.com/lostisland/faraday/blob/master/lib/faraday/request.rb#L40) ensures that the header type will be kept if you just always assign it.